module/LearnZF2Themes/src/LearnZF2Themes/Factory/ThemesFactory.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * @copyright  2015 (c) Stanimir Dimitrov.
+ * @license    http://www.opensource.org/licenses/mit-license.php  MIT License
+ *
+ * @version    0.0.18
+ *
+ * @link       TBA
```

Comment:
I'm aware of all these. Will move them later. Also I removed Themes module. Just tested it only with LearnZF2Themes module and it works fine.